lambdatimes/src/CSS/index.css

```diff
@@ -155,11 +171,15 @@ body {
   margin-left: 25px;
   flex: 1;
 }
+
+
 .header .temp {
   text-align: right;
   margin-right: 25px;
   flex: 1;
 }
+
+*/
```

Comment:
You shouldn't only comment out this code, but just go ahead and delete it! Currently, it's taking up a lot of space that it doesn't need to.